### PR TITLE
Add tests for more than 255 arguments

### PR DESCRIPTION
**Pull request by bjodah**
One limitation of `sympy.lambdify` is that it handles at most 255 arguments.
This is not an issue for `symengine.Lambdify`, this PR adds a test to ensure that this actually is the case.

### CODE REVIEW

File: symengine/tests/test_lambdify.py [hunk 1]
```diff
@@ -459,3 +460,27 @@ def test_complex_2():
...
+    # this is a proof of concept that this limitation does
+    # not affect SymEngine's Lambdify class
+    if not HAVE_NUMPY:  # nosetests work-around
+        return
+    import numpy as np
+    n = 257
+    x = se.symarray('x', n)
+    p, q, r = 17, 42, 13
+    terms = [i*s for i, s in enumerate(x, p)]
+    exprs = [reduce(add, terms), r + x[0], -99]
```
Comment (isuruf) [inline symengine/tests/test_lambdify.py]: FYI, `symengine.add(terms)` should be faster than `reduce(add, terms)`

**Comment by isuruf** [comment]
+1. Do you want add more to this PR?

File: symengine/tests/test_lambdify.py (hunk 1 above)
Comment (bjodah) [inline symengine/tests/test_lambdify.py]: It's quite strange, I get:```
  File "/home/bjorn/vc/symengine.py/symengine/tests/test_lambdify.py", line 508, in test_more_than_255_args
    exprs = [se.add(*terms), r + x[0], -99]
AttributeError: 'module' object has no attribute 'add'
```

Comment (bjodah) [inline symengine/tests/test_lambdify.py]: It was missing in __init__.py, will add it in this PR

**Comment by bjodah** [comment]
If Travis-CI tests pass I consider this PR done, feel free to merge then if there are no other comments.

**Comment by certik** [comment]
+1, thanks @bjodah .

**Pull request merged**